### DISCUSSION TITLE
500 Error code

### DISCUSSION THREAD

**Discussion by BustedSec** [Security-Onion-Solutions/securityonion#4028] [General]
Just finished running a soup update on the manager node of a distributed setup, when clicking Alerts and doing a search I get "Request failed with status code 500". Running a sudo so-status shows everything is good

Checking Docker status

    Docker ----------------------------------------------------------------------------------------------- [ OK ]

Checking container statuses

    so-cortex -------------------------------------------------------------------------------------------- [ OK ]
    so-curator ------------------------------------------------------------------------------------------- [ OK ]
    so-dockerregistry ------------------------------------------------------------------------------------ [ OK ]
    so-elastalert ---------------------------------------------------------------------------------------- [ OK ]
    so-elasticsearch ------------------------------------------------------------------------------------- [ OK ]
    so-filebeat ------------------------------------------------------------------------------------------ [ OK ]
    so-fleet --------------------------------------------------------------------------------------------- [ OK ]
    so-grafana ------------------------------------------------------------------------------------------- [ OK ]
    so-idstools ------------------------------------------------------------------------------------------ [ OK ]
    so-influxdb ------------------------------------------------------------------------------------------ [ OK ]
    so-kibana -------------------------------------------------------------------------------------------- [ OK ]
    so-kratos -------------------------------------------------------------------------------------------- [ OK ]
    so-logstash ------------------------------------------------------------------------------------------ [ OK ]
    so-mysql --------------------------------------------------------------------------------------------- [ OK ]
    so-nginx --------------------------------------------------------------------------------------------- [ OK ]
    so-playbook ------------------------------------------------------------------------------------------ [ OK ]
    so-redis --------------------------------------------------------------------------------------------- [ OK ]
    so-sensoroni ----------------------------------------------------------------------------------------- [ OK ]
    so-soc ----------------------------------------------------------------------------------------------- [ OK ]
    so-soctopus ------------------------------------------------------------------------------------------ [ OK ]
    so-telegraf ------------------------------------------------------------------------------------------ [ OK ]
    so-thehive ------------------------------------------------------------------------------------------- [ OK ]
    so-thehive-es ---------------------------------------------------------------------------------------- [ OK ]
    so-wazuh --------------------------------------------------------------------------------------------- [ OK ]

I have rebooted, and also ensured that there is plenty of free disk space on both the main system and the /nsm partition.

Please advise.

**Comment by TOoSmOotH** [comment]
Is your entire grid updated?

**Reply by marianoka** [reply]
Hi !
The problem is the nginx.conf file
The same happens to me.
